packages/xrpl/src/utils/createXchainPayment.ts

```diff
@@ -0,0 +1,40 @@
+import { XrplError } from '../errors'
+import { Payment } from '../models'
+import { Memo } from '../models/common'
+
+import { convertStringToHex } from './stringConversion'
+
+/**
+ * Computes the complete list of every balance that changed in the ledger
+ * as a result of the given transaction.
```

Comment:
Should this function work for both Mainnet -> Sidechain payments and Sidechain -> Mainnet payments?